Formula/tor.rb

```diff
@@ -39,15 +39,6 @@ def install
     system "make", "install"
   end
 
-  def caveats; <<-EOS.undent
```

Comment:
That's how you're choosing to read it, which is frustrating. My point is more "I don't expect anyone to _have_ to adhere to my concerns", whereas when I was the one handling the extreme majority of the work around formulae pre-ILZ unless someone else wanted to take that level of work on people kind of did.